egs2/iwslt22_dialect/asr1/conf/tuning/train_asr_conformer_ctc0.3_lr2e-3_warmup15k_newspecaug.yaml

```diff
@@ -0,0 +1,73 @@
+batch_type: numel
```

Comment:
Is there any reason that you keep two configs?
If so, can you add the results of both configs to README?

Reply:
Oh, I see. You already put both numbers in the README file.
Sorry that I missed this.
Please ignore my comment then.